pkg/agent/datamodel/sig_config.go

```diff
@@ -811,6 +815,13 @@ var (
 		Definition:    "windows-23H2-gen2",
 		Version:       Windows23H2SIGImageVersion,
 	}
+
+	SIGAzureLinuxV3CVMGen2ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V3cvmgen2",
```

Comment:
we don't have a successful Azlinux3+CVM VHD yet, so I don't know what definition string will be for it.

Reply:
The "Build VHD" step was successful and the string is "V3gen2CVM". Only the build test failed.

Reply:
yes, SKU_NAME=V3gen2CVM